elements/auras.lua

```diff
@@ -316,42 +359,64 @@ local function filterIcons(element, unit, filter, limit, isDebuff, offset, dontH
 end
 
 local function UpdateAuras(self, event, unit)
-	if(self.unit ~= unit) then return end
+	if self.unit ~= unit then
+		return
+	end
 
 	local auras = self.Auras
-	if(auras) then
+	if auras then
 		--[[ Callback: Auras:PreUpdate(unit)
 		Called before the element has been updated.
 
 		* self - the widget holding the aura buttons
 		* unit - the unit for which the update has been triggered (string)
 		--]]
-		if(auras.PreUpdate) then auras:PreUpdate(unit) end
+		if auras.PreUpdate then
+			auras:PreUpdate(unit)
+		end
 
 		local numBuffs = auras.numBuffs or 32
 		local numDebuffs = auras.numDebuffs or 40
 		local max = auras.numTotal or numBuffs + numDebuffs
 
-		local visibleBuffs = filterIcons(auras, unit, auras.buffFilter or auras.filter or 'HELPFUL', math.min(numBuffs, max), nil, 0, true)
+		local visibleBuffs = filterIcons(
+			auras,
+			unit,
+			auras.buffFilter or auras.filter or 'HELPFUL',
+			math.min(numBuffs, max),
+			nil,
+			0,
+			true
+		)
```

Comment:
Same problem here. I'm going to ignore further repetitions of this type of issue.

Reply:
This is also because of exceeding 120 cols. I actually like it as well.